vta/microsites
# Dynamic menu #18

This PR fixes the user experience for logging in and logging out. The menu updates based on user authentication. This functionality was aided by a free plugin, **[Nav Menu Roles](https://wordpress.org/plugins/nav-menu-roles/)**. The cart link also shows the current number of items in the customer's cart.

**Testing:**

1. Log in your account with **Login** link.
2. Navigate through different pages and see if you add an item to the cart (business card, standard-size printing, etc.).
3. Check if the Cart link updates the number i.e. `Cart (0)` ➡️ `Cart (1)`
4. On the **Cart** page, change the quantity and hit **Update Cart**. See if that also updates the cart number.
5. Log out by any method and _immediately_ navigate to the **Login** page. The WC Login page should appear (as opposed to being logged in).

**Requirements:** 

### Login
- [x] If the user is not logged in, the user should see a **Login** link and **User Registration**.
- [x] If the user is not logged in, the user should not see **My account**.
- [x] If there was a Previous login on the same browser, it does not require a manual login. This default behavior should be changed to require a manual Sign In every time.

### Log Out
- [x] The user should only see the log out button when the user is logged in.
- [x] The user should not see **User Registration** when logged in.
- [x] The user should be able to log out with a single click.
- [x] Upon logging out, the user should be redirected to the home page.

### WooCommerce Cart Update
- [x] Update the cart number with every new cart item addition
